spec/lib/active_graphql/model/relation_proxy_spec.rb

```diff
@@ -130,6 +130,20 @@ def self.name
       end
     end
 
+    describe '#select' do
+      subject(:select) { relation_proxy.select(:first_name) }
```

Comment:
can you also add more complex test case, like with `relation_proxy.select(:first_name, down: { the: 'rabbit hole' })`

Reply:
Added new test (_for rabit holes_)